io/net/virt-net/lpm.py

```diff
@@ -56,7 +58,9 @@ def setUp(self):
         self.lpar = self.get_partition_name("Partition Name")
         if not self.lpar:
             self.cancel("LPAR Name not got from lparstat command")
-
+        self.lpar_ip = self.get_mcp_component("MNName")
+        if not self.lpar_ip:
+            self.cancel("LPAR IP not got from lsrsrc command")
```

Comment:
Is this needed, since we already got lpar name from lparstat command

Reply:
@narasimhan-v the diagrmc needs lpar ip address to check for RMC connection, so we need both name and ip

Reply:
Ah okay.